src/test/java/com/jcabi/github/mock/MkHookTest.java

```diff
@@ -29,64 +29,33 @@
  */
```

Comment:
@llorllale It seems you basically rewrote all the tests that Paulo wrote [here](https://github.com/jcabi/jcabi-github/pull/1428). Why? :D

Reply:
@amihaiemil to decouple the tests.

Thanks for reminding me I was responsible for the testing structure ending up the way it was (I'm referring to the result of [that](https://github.com/jcabi/jcabi-github/pull/1428) PR): I'm too busy these days and somehow that single `MkHook` instance escaped my attention.